Framework/Kernel/inc/MantidKernel/NexusHDF5Descriptor.h

```diff
@@ -12,8 +12,7 @@
 #include <set>
 #include <string>
 
-namespace Mantid {
-namespace NeXus {
+namespace Mantid::Kernel {
```

Comment:
The Ubuntu and RHEL builders are falling over on this nested namespace definition for some reason. I have compiled locally and it is fine and I've used these elsewhere. Perhaps a clean build?

Reply:
I tried a clean build and no good. On further digging I think the problem here is actually `moc`. 
```
20:52:28 cd /home/builder/Jenkins/workspace/pull_requests-rhel7/build/qt/widgets/common/qt4/inc/MantidQtWidgets/Common && /usr/lib64/qt4/bin/moc-qt4 @/home/builder/Jenkins/workspace/pull_requests-rhel7/build/qt/widgets/common/qt4/inc/MantidQtWidgets/Common/moc_SaveWorkspaces.cxx_parameters
20:52:28 home/builder/Jenkins/workspace/pull_requests-rhel7/Framework/Kernel/inc/MantidKernel/NexusHDF5Descriptor.:15: Parse error at "Mantid"```

It looks like support for nested namespaces only came in with [Qt 5.9](https://codereview.qt-project.org/c/qt/qtbase/+/187819/) and Ubuntu/RHEL have 5.6. This explains why Windows/macOS are okay.

It looks like we can get around this by removing [this #include](https://github.com/mantidproject/mantid/blob/master/qt/widgets/common/inc/MantidQtWidgets/Common/SaveWorkspaces.h#L19) - it seems it was not required anyway.